examples/widgets/dropdown.dart

```diff
@@ -0,0 +1,57 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+class DropdownDemo extends StatefulComponent {
+  DropdownDemo();
+
+  DropdownDemoState createState() => new DropdownDemoState();
+}
+
+class DropdownDemoState extends State<DropdownDemo> {
+  dynamic _value = 0;
+
+  List <DropdownMenuItem> _buildItems() {
+    return ["One", "Two", "Free", "Four"].map((String label) {
+      return new DropdownMenuItem(value: label, child: new Text(label));
+    })
+    .toList();
+  }
+
+  Widget build(BuildContext context) {
+    Widget dropdown = new DropdownButton(
```

Comment:
if you made DropdownButton generic (DropdownButton<T>) then you could avoid having _value here be generic.